cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -367,6 +369,50 @@ func getOracleClaimType(eventType string) types.Event {
 	return claimType
 }
 
+func tryGettingCosmosBridge(
+	target common.Address,
+	client *ethclient.Client,
+	sub CosmosSub,
+	) (
+		*cosmosbridge.CosmosBridge,
+		error,
+	) {
+
+	for i := 0; i < 5; i++ {
```

Comment:
not sure if retry is useful for it, get cosmos bridge just create some structure instance, it may fail if something as abi data not found. it doesn't depend on anything like network connection. if there is a case the we can't get the bridge contract instance at first, then we can successful get one later.

Reply:
I believe this call to NewCosmosBridge does require a network connection as it takes an ethereum client as a parameter.

Reply:
Removed.